src/forms/BookingDatesForm/EstimatedBreakdownMaybe.js

```diff
@@ -87,18 +91,9 @@ const estimatedTransaction = (unitType, bookingStart, bookingEnd, unitPrice, qua
       createdAt: now,
       lastTransitionedAt: now,
       lastTransition: TRANSITION_REQUEST_PAYMENT,
-      payinTotal: totalPrice,
-      payoutTotal: totalPrice,
-      lineItems: [
-        {
-          code: unitType,
-          includeFor: ['customer', 'provider'],
-          unitPrice: unitPrice,
-          quantity: new Decimal(unitCount),
-          lineTotal: totalPrice,
-          reversal: false,
-        },
-      ],
+      payinTotal,
+      payoutTotal,
+      lineItems: customerLineItems,
```

Comment:
Before this component was for both customer and provide, although it's only ever shown for customers in the listing page. However, now the customer line items are only used, so maybe at least add a clear comment that the estimation breakdown is for the customer. Possibly even rename the component?

Reply:
I decided to add customerRole variable there so that it would be easier to use the same component also for the provider if needed in the future or in some customization.